packages/cli-exec/test/exec.test.js

```diff
@@ -173,6 +173,25 @@ describe('percy exec', () => {
     });
   });
 
+  it('tests process.stderr percy is disabled', async () => {
+    delete process.env.PERCY_CLIENT_ERROR_LOGS;
+    process.env.PERCY_ENABLE = '0';
```

Comment:
this is not reset at the end of the test

Reply:
There is a after each block that deletes this delete process.env.PERCY_ENABLE;